### ISSUE TITLE
Enterprise Security in Docker container

### ISSUE THREAD

**Issue by yampelo** [splunk/docker-splunk-legacy#66]
Is it possible to run ES in a docker container? I've tried and it breaks the container

**Comment by halr9000** [comment]
@yampelo Yes, but note that the ES (and ITSI) system requirements are higher than core. For example when running ITSI in Docker on macOS, I had to increase the size of the Hyeprkit VM which backs Docker. You may or may not be running into a similar issue. 

If you want to keep this issue open, please add details to address some of the areas mentioned above. thanks
